pkg/aws/helpers.go

```diff
@@ -400,3 +408,24 @@ func GenerateOperatorPolicyFiles(reporter *rprtr.Object) error {
 
 	return nil
 }
+
+func GenerateUserRolePolicyFiles(reporter *rprtr.Object, env string, accountID string) error {
+	filename := fmt.Sprintf("sts_%s_trust_policy.json", OCMUserRolePolicyFile)
+	path := fmt.Sprintf("templates/policies/%s", filename)
+
+	policy, err := ReadPolicyDocument(path, map[string]string{
+		"aws_account_id": JumpAccounts[env],
+		"ocm_account_id": accountID,
+	})
+	if err != nil {
+		return err
+	}
+
+	reporter.Debugf("Saving '%s' to the current directory", filename)
+	err = saveDocument(policy, filename)
```

Comment:
I'm not entirely sure I like this function being called here, rather than in the `cmd` file. This is a lot of responsability for a 'helper' file.